survey_odoo_debranding/templates/disable_odoo.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <template
+        id="layout_footer_copyright"
+        inherit_id="web.brand_promotion_message"
```

Comment:
This template is used in more places than survey and you don't even have a dependency to survey. Maybe it would be good to describe where else it applies and put a more generic name?